src/algebra/order/hom/ring.lean

```diff
@@ -277,6 +278,9 @@ ext e.left_inv
 @[simp] lemma symm_trans_self (e : α ≃+*o β) : e.symm.trans e = order_ring_iso.refl β :=
 ext e.right_inv
 
+lemma symm_injective : injective (order_ring_iso.symm : (α ≃+*o β) → β ≃+*o α) :=
+λ f g h, f.symm_symm.symm.trans $ (congr_arg order_ring_iso.symm h).trans g.symm_symm
```

Comment:
Normally we state this as `symm_bijective` instead, see `linear_equiv.symm_bijective` etc.